src/CommunityToolkit/Xamarin.CommunityToolkit/Views/MediaElement/Tizen/MediaPlayer.tizen.cs

```diff
@@ -0,0 +1,563 @@
+
+		public static readonly BindableProperty AutoStopProperty = BindableProperty.Create(nameof(AutoStop), typeof(bool), typeof(MediaPlayer), true, propertyChanged: (b, o, n) => ((MediaPlayer)b).UpdateAutoStop());
+
+		public static readonly BindableProperty IsLoopingProperty = BindableProperty.Create(nameof(IsLooping), typeof(bool), typeof(MediaPlayer), false, propertyChanged: (b, o, n) => ((MediaPlayer)b).UpdateIsLooping());
+
+		static readonly BindablePropertyKey DurationPropertyKey = BindableProperty.CreateReadOnly(nameof(Duration), typeof(int), typeof(MediaPlayer), 0);
+
+		public static readonly BindableProperty DurationProperty = DurationPropertyKey.BindableProperty;
+
+		static readonly BindablePropertyKey BufferingProgressPropertyKey = BindableProperty.CreateReadOnly(nameof(BufferingProgress), typeof(double), typeof(MediaPlayer), 0d);
+
+		public static readonly BindableProperty BufferingProgressProperty = BufferingProgressPropertyKey.BindableProperty;
+
+		static readonly BindablePropertyKey PositionPropertyKey = BindableProperty.CreateReadOnly(nameof(Position), typeof(int), typeof(MediaPlayer), 0);
+
+		public static readonly BindableProperty PositionProperty = PositionPropertyKey.BindableProperty;
+
+		static readonly BindablePropertyKey StatePropertyKey = BindableProperty.CreateReadOnly(nameof(State), typeof(PlaybackState), typeof(MediaPlayer), PlaybackState.Stopped);
+
+		public static readonly BindableProperty StateProperty = StatePropertyKey.BindableProperty;
+
+		public static readonly BindableProperty PositionUpdateIntervalProperty = BindableProperty.Create(nameof(PositionUpdateInterval), typeof(int), typeof(MediaPlayer), 200);
+
+		static readonly BindablePropertyKey IsBufferingPropertyKey = BindableProperty.CreateReadOnly(nameof(IsBuffering), typeof(bool), typeof(MediaPlayer), false);
+
+		public static readonly BindableProperty IsBufferingProperty = IsBufferingPropertyKey.BindableProperty;
+
+		bool disposed = false;
+		bool isDisposing = false;
+		bool isPlaying;
+		bool controlsAlwaysVisible;
+		IPlatformMediaPlayer impl;
+		CancellationTokenSource hideTimerCTS = new CancellationTokenSource();
+		Lazy<View> controls;
+
+		public MediaPlayer()
+		{
+			impl = new MediaPlayerImpl();
+			impl.UpdateStreamInfo += OnUpdateStreamInfo;
+			impl.PlaybackCompleted += SendPlaybackCompleted;
+			impl.PlaybackStarted += SendPlaybackStarted;
+			impl.PlaybackPaused += SendPlaybackPaused;
+			impl.PlaybackStopped += SendPlaybackStopped;
+			impl.BufferingProgressUpdated += OnUpdateBufferingProgress;
+			impl.ErrorOccurred += OnErrorOccurred;
+			impl.UsesEmbeddingControls = true;
+			impl.Volume = 1d;
+			impl.AspectMode = DisplayAspectMode.AspectFit;
+			impl.AutoPlay = false;
+			impl.AutoStop = true;
+
+			controlsAlwaysVisible = false;
+			controls = new Lazy<View>(() =>
+			{
+				return impl.GetEmbeddingControlView(this);
+			});
+		}
+
+		~MediaPlayer()
```

Comment:
From Xamarin.Forms controls implementations we don't implement `IDispose` in xaml controls. Is it really needed here? Why Tizen can't use the `MediaElement.shared.cs`?